compiler/rustc_hir/src/intravisit.rs

```diff
@@ -1197,6 +1199,7 @@ pub fn walk_struct_def<'v, V: Visitor<'v>>(
 pub fn walk_field_def<'v, V: Visitor<'v>>(visitor: &mut V, field: &'v FieldDef<'v>) -> V::Result {
```

Comment:
Could you use a destructuring assignment so it errs on changes?

Reply:
done